java/code/src/com/suse/manager/webui/utils/LoginHelper.java

```diff
@@ -65,8 +66,13 @@
 
     private static Logger log = Logger.getLogger(LoginHelper.class);
     private static final String DEFAULT_KERB_USER_PASSWORD = "0";
-    private static final Long MIN_PG_DB_VERSION = 90600L;
-    private static final String MIN_PG_DB_VERSION_STRING = "9.6";
+    private static final Long MIN_PG_DB_VERSION = 100001L;
+    private static final Long MAX_PG_DB_VERSION = 130000L;
+    private static final String MIN_PG_DB_VERSION_STRING = "10";
+    private static final String MAX_PG_DB_VERSION_STRING = "12";
```

Comment:
Here it is 12, but the Long is 130000 . Is this intentional?

Reply:
yes as I check for "less than". But we could also set 129999L and test for "less or equal than"

Reply:
You were right. I checked already for greater than which means we should test for 129999L value. Changed.